splunk_connector.py

```diff
@@ -944,6 +950,7 @@ def _on_poll(self, param):  # noqa: C901
                         name_mappings[k.lower()] = k
                 for h in header_set:
                     cef_name = consts.CIM_CEF_MAP.get(h, h)
```

Comment:
Why not remove this?

Reply:
Here, CIM_CEF_MAP has a mapping for all the CEF keys as they are case sensitive. So, I am not sure, why this needs to be removed.

Reply:
My bad I misread